### PR TITLE
[4.13-4.14] migrate some ovn/sdn migration jobs to multi-arch ci

### PR DESCRIPTION
**Pull request by jianlinliu**
Follow up https://github.com/openshift/release/pull/44265

### CODE REVIEW

**Comment by jianlinliu** [comment]
cc @zhaozhanqi and @yunjiang29 to review.

File: ci-operator/config/openshift/openshift-tests-private/openshift-openshift-tests-private-release-4.13__multi-nightly.yaml [hunk 1]
```diff
@@ -79,6 +79,97 @@ tests:
...
+    cluster_profile: aws-qe
+    dependencies:
+      OPENSHIFT_INSTALL_RELEASE_IMAGE_OVERRIDE: release:multi-latest
+    env:
+      BASE_DOMAIN: qe.devcluster.openshift.com
+      COMPUTE_NODE_TYPE: m6g.xlarge
+      CONTROL_PLANE_INSTANCE_TYPE: m6g.xlarge
+      OCP_ARCH: arm64
+      UPGRADE_CHECK_RUN_TAGS: '@arm64 and @4.13 and @aws-ipi and @upgrade-check and
+        @network-ovnkubernetes'
```
Comment (zhaozhanqi) [inline ci-operator/config/openshift/openshift-tests-private/openshift-openshift-tests-private-release-4.13__multi-nightly.yaml:142]:
```suggestion
        @network-ovnkubernetes and not @fips'
```
Could you also help fix this one, it's should be one potential issue.

File: ci-operator/config/openshift/openshift-tests-private/openshift-openshift-tests-private-release-4.13__multi-nightly.yaml [hunk 2]
```diff
@@ -79,6 +79,97 @@ tests:
...
+      UPGRADE_PRE_RUN_TAGS: '@arm64 and @4.13 and @aws-ipi and @upgrade-prepare and
+        @network-openshiftsdn'
```
Comment (zhaozhanqi) [inline ci-operator/config/openshift/openshift-tests-private/openshift-openshift-tests-private-release-4.13__multi-nightly.yaml:144]:
```suggestion
        @network-openshiftsdn and not @fips'
```

File: ci-operator/config/openshift/openshift-tests-private/openshift-openshift-tests-private-release-4.14__multi-nightly.yaml [hunk 3]
```diff
@@ -189,6 +189,122 @@ tests:
...
+    cluster_profile: aws-qe
+    dependencies:
+      OPENSHIFT_INSTALL_RELEASE_IMAGE_OVERRIDE: release:multi-latest
+    env:
+      BASE_DOMAIN: qe.devcluster.openshift.com
+      COMPUTE_NODE_TYPE: m6g.xlarge
+      CONTROL_PLANE_INSTANCE_TYPE: m6g.xlarge
+      OCP_ARCH: arm64
+      UPGRADE_CHECK_RUN_TAGS: '@arm64 and @4.14 and @aws-ipi and @upgrade-check and
+        @network-ovnkubernetes'
```
Comment (zhaozhanqi) [inline ci-operator/config/openshift/openshift-tests-private/openshift-openshift-tests-private-release-4.14__multi-nightly.yaml:277]:
```suggestion
        @network-ovnkubernetes and not @fips'
```

File: ci-operator/config/openshift/openshift-tests-private/openshift-openshift-tests-private-release-4.14__multi-nightly.yaml [hunk 4]
```diff
@@ -189,6 +189,122 @@ tests:
...
+      UPGRADE_PRE_RUN_TAGS: '@arm64 and @4.14 and @aws-ipi and @upgrade-prepare and
+        @network-openshiftsdn'
```
Comment (zhaozhanqi) [inline ci-operator/config/openshift/openshift-tests-private/openshift-openshift-tests-private-release-4.14__multi-nightly.yaml:279]:
```suggestion
        @network-openshiftsdn and not @fips'
```

**Comment by zhaozhanqi** [comment]
cc @anuragthehatter @weliang1 ^^

File: ci-operator/config/openshift/openshift-tests-private/openshift-openshift-tests-private-release-4.14__multi-nightly.yaml (hunk 4 above)
Comment (jianlinliu) [reply ci-operator/config/openshift/openshift-tests-private/openshift-openshift-tests-private-release-4.14__multi-nightly.yaml]: fixed

File: ci-operator/config/openshift/openshift-tests-private/openshift-openshift-tests-private-release-4.14__multi-nightly.yaml (hunk 3 above)
Comment (jianlinliu) [reply ci-operator/config/openshift/openshift-tests-private/openshift-openshift-tests-private-release-4.14__multi-nightly.yaml]: fixed

File: ci-operator/config/openshift/openshift-tests-private/openshift-openshift-tests-private-release-4.13__multi-nightly.yaml (hunk 2 above)
Comment (jianlinliu) [reply ci-operator/config/openshift/openshift-tests-private/openshift-openshift-tests-private-release-4.13__multi-nightly.yaml]: fixed

File: ci-operator/config/openshift/openshift-tests-private/openshift-openshift-tests-private-release-4.13__multi-nightly.yaml (hunk 1 above)
Comment (jianlinliu) [reply ci-operator/config/openshift/openshift-tests-private/openshift-openshift-tests-private-release-4.13__multi-nightly.yaml]: fixed

**Comment by jianlinliu** [comment]
cc @liangxia to review.

**Pull request merged**